Makefile

```diff
@@ -91,11 +93,15 @@ LIBMCOUNT_TARGETS += libmcount/libmcount-single.so libmcount/libmcount-fast-sing
 
 _TARGETS := uftrace libtraceevent/libtraceevent.a
 _TARGETS += $(LIBMCOUNT_TARGETS) libmcount/libmcount-nop.so
+_TARGETS += misc/demangler
 TARGETS  := $(patsubst %,$(objdir)/%,$(_TARGETS))
 
 UFTRACE_SRCS := $(srcdir)/uftrace.c $(wildcard $(srcdir)/cmd-*.c $(srcdir)/utils/*.c)
 UFTRACE_OBJS := $(patsubst $(srcdir)/%.c,$(objdir)/%.o,$(UFTRACE_SRCS))
 
+DEMANGLER_SRCS := $(srcdir)/misc/demangler.c $(srcdir)/utils/demangle.c $(srcdir)/utils/debug.c
+DEMANGLER_OBJS := $(patsubst $(srcdir)/misc/%.c,$(objdir)/misc/%.o,$(DEMANGLER_SRCS))
```

Comment:
No, it won't match `$(srcdir)/utils/*` anymore.  You need to remove the `/misc` part IMO.

Reply:
Like this way?
```diff
diff --git a/Makefile b/Makefile
index 8e63ff6..805bfed 100644
--- a/Makefile
+++ b/Makefile
@@ -100,7 +100,7 @@ UFTRACE_SRCS := $(srcdir)/uftrace.c $(wildcard $(srcdir)/cmd-*.c $(srcdir)/utils
 UFTRACE_OBJS := $(patsubst $(srcdir)/%.c,$(objdir)/%.o,$(UFTRACE_SRCS))
 
 DEMANGLER_SRCS := $(srcdir)/misc/demangler.c $(srcdir)/utils/demangle.c $(srcdir)/utils/debug.c
-DEMANGLER_OBJS := $(patsubst $(srcdir)/misc/%.c,$(objdir)/misc/%.o,$(DEMANGLER_SRCS))
+DEMANGLER_OBJS := $(patsubst $(srcdir)/%.c,$(objdir)/%.o,$(DEMANGLER_SRCS))
 
 UFTRACE_ARCH_OBJS := $(objdir)/arch/$(ARCH)/uftrace.o 
```